core/src/code.rs

```diff
@@ -69,6 +77,34 @@ fn get_exports(
         }
     }
 
+    if reject_unnecessary {
+        let funcs = module
+            .function_section()
+            .ok_or(CodeError::FunctionSectionNotFound)?
+            .entries();
+
+        let types = module
+            .type_section()
+            .ok_or(CodeError::TypeSectionNotFound)?
+            .types();
+
+        let import_count = module
+            .import_section()
+            .ok_or(CodeError::ImportSectionNotFound)?
+            .entries()
```

Comment:
just use `ImportSection::functions` method, not `entries`.

Reply:
Wait nevermind that is exactly what I need, sorry.